packages/components/src/navigator/navigator-button/README.md

```diff
@@ -35,4 +31,4 @@ The path of the screen to navigate to. The value of this prop needs to be [a val
 
 ### Inherited props
 
-`NavigatorButton` also inherits all of the [`Button` props](/packages/components/src/button/README.md#props), except for `href` and `target`.
+`Navigator.Button` also inherits all of the [`Button` props](/packages/components/src/button/README.md#props), except for `href` and `target`.
```

Comment:
I'm not 100% sure if we should refer to the components in the docs with their namespaced display names or their original names. My take atm is that since this will be read by the consumers, using the namespaced name seems right, and the original constant becomes an implementation detail.

Reply:
Yup this seems reasonable to me